naptime-graphql/src/test/scala/org/coursera/naptime/ari/graphql/helpers/ArgumentBuilder.scala

```diff
@@ -0,0 +1,11 @@
+package org.coursera.naptime.ari.graphql.helpers
+
+import sangria.schema.Args
+
+import scala.collection.concurrent.TrieMap
+
+object ArgumentBuilder {
+  def buildArgs(args: Map[String, Any] = Map.empty, limit: Int = 10, start: Option[Int] = None): Args = {
```

Comment:
can we modify this to take in the following signature:
```lang=scala
def buildArgs(arguments: List[Argument[_]]): Args = ???
```
and then use the default values / inputTypes on the argument to determine how to build the Args (instead of hardcoding `limit` and `start` as arguments?

let me know if you have questions about this!